### PR TITLE
brought to Play 2.4

### PR DESCRIPTION
**Pull request by akauppi**
Changes to bring the project to Play 2.4.
- compiles and the 'positive' tests pass
- 404 tests fail, haven't looked into what's changed in 2.4 concerning them

Would be delighted if you check this out, and maybe take as a separate branch into your main repo. I'm not suggesting the master necessarily needs to be brought to Play 2.4, yet.

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import sbt._
-import Keys._
+import sbt.Keys._
```
Comment (akauppi) [inline project/Build.scala]: this change seems unintentional (blaiming the IDE)

**Comment by andrewconner** [comment]
Awesome, thanks! I'll take a look later today.

**Comment by andrewconner** [comment]
Many apologies, this is the longest "later today" you've ever experienced. Merging to main branch.

**Comment by mslinn** [comment]
The README says "A modular Play 2.3.6 application using multi-projects". You might want to change it to read "A modular Play 2.4.3 application using multi-projects". Any thoughts on upgrading this to 2.5.3?

**Pull request merged**